src/test/java/org/opensearch/knn/index/OpenSearchIT.java

```diff
@@ -263,6 +265,35 @@ public void testIndexingVectorValidation_differentSizes() throws Exception {
         assertThat(EntityUtils.toString(ex.getResponse().getEntity()), containsString("Vector dimension mismatch. Expected: 4, Given: 5"));
     }
 
+    @SneakyThrows
+    public void testIndexingVectorValidation_zeroVector() {
```

Comment:
This test should be part of Either Lucene IT or IndexingIT. This class is a class for containing common functions. Lets not add tests where which runs.

Reply:
I think this test is similar to `OpenSearchIT#testIndexingVectorValidation_differentSizes`, and it is not limited to the Lucene engine, but can also be applied to other engines.